lib/node_modules/@stdlib/stats/base/dists/logistic/cdf/benchmark/benchmark.js

```diff
@@ -33,16 +34,24 @@ var cdf = require( './../lib' );
 bench( pkg, function benchmark( b ) {
 	var mu;
 	var s;
+	var len;
```

Comment:
```suggestion
	var len;
	var mu;
	var s;
```

Variables are declared according to decreasing order of their length